README.md

```diff
@@ -154,9 +167,10 @@ bookkeeping system using plaintext files.
 
 ## Price Sources
 
-- [xuhcc/beancount-exchangerates](https://github.com/xuhcc/beancount-exchangerates) for [exchangerate.host](https://exchangerate.host/) or [frankfurter.dev](https://frankfurter.dev/) or [exchangeratesapi.io](https://exchangeratesapi.io/) or other [fixer.io](https://fixer.io/)-like 💱 APIs
+- [xuhcc/beancount-exchangerates](https://github.com/xuhcc/beancount-exchangerates) for [frankfurter.dev](https://frankfurter.dev/) or [exchangeratesapi.io](https://exchangeratesapi.io/) or other [fixer.io](https://fixer.io/)-like 💱 APIs
```

Comment:
It looks like `https://exchangerate.host` was removed. Let's add it back, unless there's a specific reason for the removal?

Reply:
@siddhantgoel yeah this was intentional, remember I added that in #57 but then in https://github.com/xuhcc/beancount-exchangerates/pull/5 (and https://github.com/xuhcc/beancount-exchangerates/commit/4f25d64f9d0d439f589a7289decaad8fa1d853ff) we've clarified that this price importer actually no longer supports that https://exchangerate.host as source. Apologies that I "threw this in here", just for convenience, instead of more clearly raising a separate PR to clarify that. OK for you to merge as-is?

Reply:
No worries, sure let's merge as is.